src/version-selector.js

```diff
@@ -64,7 +66,17 @@
     window.openVersion = function () {
       var version = selectTemplate.options[selectTemplate.selectedIndex].value;
 
-      window.location.href = '/' + version;
+      window.location.href = '/' + version + '/' + currentComponent;
     };
   }
+
+  function getCurrentVersion() {
+    var selectedVersion = urlArray[1];
+    return selectedVersion;
+  }
+
+  function getCurrentComponent() {
```

Comment:
Deze functie, samen met `getCurrentVersion()` wil ik graag "pure" hebben: http://www.nicoespeon.com/en/2015/01/pure-functions-javascript/

Beide functies accepteren 1 argument: `pathname`. Bovenaan deze file worden ze beide aangeroepen met `window.location.pathname`.

Reply:
Ja, dat is wel een stuk prettiger